source/Assets/Plugins/Didomi/IOS/Didomi.mm

```diff
@@ -736,6 +813,12 @@ void addEventListener( void (*event_listener_handler) (DDMEventType, NSString *
 
     };
 
+	eventListener.onError = ^(DDMErrorEvent * errorEvent){
+
+        event_listener_handler(1000 , errorEvent.descriptionText);
```

Comment:
what does this 1000 mean?

Reply:
To bridge it between the IOS SDK and Unity SDK, type of the input changed from DDMEventType to the int. And the value 1000 used for error event type value.

The code changed like below for making it clear.
        int errorEventEnumValue = 1000;
        event_listener_handler(errorEventEnumValue , errorEvent.descriptionText);